File: app/src/main/java/np/com/aawaz/csitentrance/objects/Feedback.java
package np.com.aawaz.csitentrance.objects;

import com.google.firebase.auth.FirebaseAuth;
import com.google.firebase.auth.FirebaseUser;
import com.google.firebase.database.IgnoreExtraProperties;

@IgnoreExtraProperties
public class Feedback {

    public String name;
    public String email;
    public String message;
    public String uuid;

    public Feedback() {
        // Default constructor required for calls to DataSnapshot.getValue(Post.class)
    }

    public Feedback(String message) {
        FirebaseUser user = FirebaseAuth.getInstance().getCurrentUser();
        this.name = user.getDisplayName();
        this.email = user.getEmail();
        this.message = message;
        this.uuid = user.getUid();
    }

}

File: app/src/main/java/np/com/aawaz/csitentrance/objects/Feedback.kt
package np.com.aawaz.csitentrance.objects

import com.google.firebase.auth.FirebaseAuth
import com.google.firebase.auth.FirebaseUser
import com.google.firebase.database.IgnoreExtraProperties

@IgnoreExtraProperties
class Feedback(var message: String) {

    var name: String? = null
    var email: String? = null
    var uuid: String

    init {
        val user = FirebaseAuth.getInstance().currentUser
        this.name = user!!.displayName
        this.email = user.email
        this.uuid = user.uid
    }

}
